python/lsst/rucio/register/script.py

```diff
@@ -47,14 +55,11 @@ def chunks(refs, chunk_size):
         yield itertools.chain((start,), chunk)
 
 
-def main():
-    """Command line interface for rucio-register"""
-    ap = AppParser(sys.argv)
-
+def getRucioInterface(repo, rucio_register_config, rubin_butler_type):
```

Comment:
It looks like a public function, please add a docstring or make it private.

Reply:
Fixed